lib/api/include/fileChksum.h

```diff
@@ -1,25 +1,51 @@
 #ifndef FILE_CHKSUM_H__
 #define FILE_CHKSUM_H__
 
-#include "rcConnect.h"
+/// \file
+
 #include "rodsDef.h"
+#include "objInfo.h"
+#include "rodsType.h"
+
+struct RcComm;
 
 typedef struct FileChksumInp {
     rodsHostAddr_t addr;
     char fileName[MAX_NAME_LEN];
     char rescHier[MAX_NAME_LEN];
     char objPath[MAX_NAME_LEN];
-    int flag;   // not used for now
-    char in_pdmo[MAX_NAME_LEN]; // Flag indicating if we are being executed from a pdmo
-    char orig_chksum[NAME_LEN]; // original incoming checksum
+    int flag;                   // Not used for now.
+    char in_pdmo[MAX_NAME_LEN]; // Flag indicating if we are being executed from a pdmo.
+    char orig_chksum[NAME_LEN]; // Original incoming checksum.
+    rodsLong_t dataSize;
```

Comment:
dataSize is added here, but not in the #define a few lines below

Reply:
updated packing instruction